Contents/Code/__init__.py

```diff
@@ -152,14 +152,14 @@ def Update(self, metadata, media, lang, force, movie):
 
                 if len(series['art']['thumb']):
                     for art in series['art']['thumb']:
-                        episodeObj.thumbs[art['url']] = Proxy.Media(HTTP.Request(art['url']).content, art['index'])
+                        episodeObj.thumbs[art['url']] = Proxy.Media(HTTP.Request("http://{host}:{port}{relativeURL}".format(host=Prefs['Hostname'], port=Prefs['Port'], relativeURL=art['url'])).content, art['index'])
```

Comment:
I used the newer [PEP 3101 style string formatting](https://www.python.org/dev/peps/pep-3101/), although it was introduced with Python 3 it was backported to Python 2.7 and it's a bit of a habit of mine to use the newer style - I find it a little easier to read and understand what variable goes where in the string too. If I make any future commits would you prefer me to use the old style or new style?

Thanks for merging 😄